framework/src/wifi_manager/wifi_manager.c

```diff
@@ -541,8 +541,12 @@ _convert_scan_info(wifi_manager_scan_info_s **wm_scan_list, wifi_utils_scan_list
 		cur->rssi = iter->ap_info.rssi;
 		cur->channel = iter->ap_info.channel;
 		cur->phy_mode = iter->ap_info.phy_mode;
-		strncpy(cur->ssid, (char *)iter->ap_info.ssid, 32);
-		strncpy(cur->bssid, (char *)iter->ap_info.bssid, 17);
+		cur->ap_auth_type = iter->ap_info.ap_auth_type;
+		cur->ap_crypto_type = iter->ap_info.ap_crypto_type;
+		strncpy(cur->ssid, (char *)iter->ap_info.ssid, WIFIMGR_SSID_LEN);
+		cur->ssid[WIFIMGR_SSID_LEN-1] = '\0';
+		strncpy(cur->bssid, (char *)iter->ap_info.bssid, WIFIMGR_MACADDR_STR_LEN);
+		cur->bssid[WIFIMGR_MACADDR_STR_LEN-1] = '\0';
```

Comment:
same as previous

Reply:
Thanks for comment. Modification completed.